resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -231,7 +248,115 @@ class MousePointer {
+    [DscProperty()] [BinarySettingState] $ShowShadowsUnderMousePointer = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOrSlideToolTipsIntoView = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $FadeOutMenuItemsAfterClicking = [BinarySettingState]::KeepCurrentValue
+    [DscProperty()] [BinarySettingState] $ShowShadowsUnderWindows = [BinarySettingState]::KeepCurrentValue
+    [DscProperty(NotConfigurable)] [AnimationEffects] $currentState
+
+    [AnimationEffects] Get() {
+        $this.currentState = [AnimationEffects]::new()
+
+		$this.currentState = [BinarySettingState]::Disabled
+
+		foreach ($enum in [AnimationEffectsSettings].GetEnumNames()) {
+			$thisState = Get-AnimationState $enum
+			if ($thisState -eq [BinarySettingState]::Enabled) {
+				$this.currentState = [BinarySettingState]::Enabled
+			}
+		}
+
+		return $this.currentState
+	}
+
+    [bool] Test() {
+		$this.currentState = $this.Get()
+		if ($this.SmoothScrollListBoxes -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.SmoothScrollListBoxes -ne $this.currentState.SmoothScrollListBoxes) {
+
+			return $false
+		}
+		
+		else if ($this.SlideOpenComboBoxes -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.SlideOpenComboBoxes -ne $this.currentState.SlideOpenComboBoxes) {
+
+			return $false
+		}
+		
+		else if ($this.FadeOrSlideMenusIntoView -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.FadeOrSlideMenusIntoView -ne $this.currentState.FadeOrSlideMenusIntoView) {
+
+			return $false
+		}
+		
+		else if ($this.ShowShadowsUnderMousePointer -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.ShowShadowsUnderMousePointer -ne $this.currentState.ShowShadowsUnderMousePointer) {
+
+			return $false
+		}
+		
+		else if ($this.ShowShadowsUnderMousePointer -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.ShowShadowsUnderMousePointer -ne $this.currentState.ShowShadowsUnderMousePointer) {
+
+			return $false
+		}
+		
+		else if ($this.FadeOrSlideToolTipsIntoView -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.FadeOrSlideToolTipsIntoView -ne $this.currentState.FadeOrSlideToolTipsIntoView) {
+
+			return $false
+		}
+		
+		else if ($this.FadeOutMenuItemsAfterClicking -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.FadeOutMenuItemsAfterClicking -ne $this.currentState.FadeOutMenuItemsAfterClicking) {
+
+			return $false
+		}
+		
+		else if ($this.ShowShadowsUnderWindows -ne [BinarySettingState]::KeepCurrentValue -or
+		$this.ShowShadowsUnderWindows -ne $this.currentState.ShowShadowsUnderWindows) {
+
+			return $false
+		} else {
+			
+		return $true
+		}
+    }
+
+    [void] Set() {
+		if ($this.Test() -eq $false) {
+            $SmoothScrollListBoxesDesiredValue = switch ([BinarySettingState]($this.SmoothScrollListBoxes)) {
+                Enabled {1}
+                Disabled {0}
+            }

```

Comment:
Why is this property special cased?

Reply:
It was a placeholder for the AnimationEffects variable. This has been removed.